src/ui2d.js

```diff
@@ -16,9 +16,9 @@ window.addEventListener('load', function (event) {
     instructions.classList.remove('hide');
   }
 
-  sceneEl.addEventListener('enter-vr', function () {
-    instructions.classList.add('hide');
-  });
+  // sceneEl.addEventListener('enter-vr', function () {
```

Comment:
can you remove this change? so it won't conflict with https://github.com/aframevr/a-saturday-night/pull/120